### PR TITLE
v1.5.0: Add Countries resource and User.mobile_allow_live_timetracking field

### PR DESCRIPTION
**Pull request by AlGvrl**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -102,11 +104,14 @@ export default class Api {
...
   public absenceTypes: AbsenceTypesEndpoint;
   public authentication: AuthenticationEndpoint;
+  public countries: CountriesEndpoint;
   public deltaSync: DeltaSyncEndpoint;
   public departments: DepartmentsEndpoint;
   public favouriteTasks: FavouriteTasksEndpoint;
   public feedback: FeedbackEndpoint;
   public generalSettings: GeneralSettingsEndpoint;
+  public healthRules: HealthRulesEndpoint;
+  public healthData: HealthDataEndpoint;
```
Comment (AlGvrl) [inline src/index.ts:114]: Moved these two just for the alphabetical order

File: src/countries/countries.test.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { expect } from '@jest/globals';
+import axios from 'axios';
+
+import AxiosMockAdapter from 'axios-mock-adapter';
+import { ConfigProvider } from '../utils';
+import { LibraryReturn } from '../utils/response/apiResponse';
+import { Country } from './types';
+import { CountriesEndpoint } from './';
+
+const MockData: {
```
Comment (benj56) [inline src/countries/countries.test.ts:10]:
```suggestion
const mockData: {
```

**Comment by tomi-timetac** [comment]
@AlGvrl Update `README` to flag the countries endpoint as done.

File: src/countries/countries.test.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    expect.assertions(1);
+
+    await countries.read().then((results: LibraryReturn<'countries', Country[]>) => {
+      expect(results).toStrictEqual({
+        Results: MockData.countriesReadResult,
+        Affected: {},
+        Deleted: [],
+      });
+    });
```
Comment (benj56) [inline src/countries/countries.test.ts:50]:
```suggestion
    const results = await countries.read();
    expect(results).toStrictEqual({
        Results: MockData.countriesReadResult,
        Affected: {},
        Deleted: [],
    });
```

**Review by benj56 — APPROVED** [review]

File: src/countries/countries.test.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  const readPath = `${countries.getResourcePath()}/read`;
+
+  const mock = new AxiosMockAdapter(axios);
+
+  afterEach(() => {
+    mock.reset();
+  });
+
+  test('read', async () => {
+    mock.onGet(readPath).reply(200, { Success: true, NumResults: 3, Results: MockData.countriesReadResult });
```
Comment (AlGvrl) [inline src/countries/countries.test.ts:40]:
```suggestion
    mock.onGet(readPath).reply(200, { Success: true, NumResults: 3, Results: mockData.countriesReadResult });
```

File: src/countries/countries.test.ts [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+    expect.assertions(1);
+
+    await countries.read().then((results: LibraryReturn<'countries', Country[]>) => {
+      expect(results).toStrictEqual({
+        Results: MockData.countriesReadResult,
```
Comment (AlGvrl) [inline src/countries/countries.test.ts:46]:
```suggestion
        Results: mockData.countriesReadResult,
```

**Review by benj56 — APPROVED** [review]

**Pull request merged**